src/model/BridgeGame.js

```diff
@@ -0,0 +1,70 @@
+const Validation = require('../utils/Validation');
+const {
+  ONE_TIME,
+  UP,
+  DOWN,
+  CROSSED,
+  BLOCKED,
+  BLANK,
+  ZERO,
+} = require('../utils/constants');
+
+class BridgeGame {
+  #retryingCount = ONE_TIME;
+  #upperBridge = [];
+  #lowerBridge = [];
+
+  static validateDirection(direction) {
+    Validation.checkBlank(direction);
+    Validation.checkStringType(direction);
+    Validation.checkUpperCaseOfDirection(direction);
+    Validation.checkValidDirection(direction);
+  }
+
+  canMove(direction, winningBridge) {
+    return winningBridge.isSameDirection(direction, this.#upperBridge.length);
+  }
+
+  move(direction, CAN_MOVE) {
+    if (direction === UP) {
+      this.#upperBridge.push(CAN_MOVE ? CROSSED : BLOCKED);
+      this.#lowerBridge.push(BLANK);
+    }
+
+    if (direction === DOWN) {
+      this.#lowerBridge.push(CAN_MOVE ? CROSSED : BLOCKED);
+      this.#upperBridge.push(BLANK);
+    }
+  }
+
+  isLastStage(winningBridge) {
+    return winningBridge.isSameLength(this.#upperBridge);
+  }
+
+  static validateCommand(command) {
+    Validation.checkBlank(command);
+    Validation.checkStringType(command);
+    Validation.checkUpperCaseOfCommand(command);
+    Validation.checkValidCommand(command);
+  }
+
+  getUpperBridge() {
+    return this.#upperBridge;
+  }
+
+  getLowerBridge() {
+    return this.#lowerBridge;
+  }
```

Comment:
```suggestion
  getBridge() {
    return {
      upperbridge: this.#upperBridge,
      lowerbridge: this.#lowerBridge,
    }
  }
```
`getBridge` 메서드가 객체를 반환하고 메서드를 사용하는 쪽에서 객체 구조분해할당으로 사용하는 것도 좋을 것 같습니다..!

Reply:
앗 네 그런 방법이 있네요!! 생각하지 못했는데 좋은 아이디어 감사합니다 😊👍